### ISSUE TITLE
Bug: (fill in)

### ISSUE THREAD

**Issue by arnotixe** [eslint/eslint#16910]
### Environment

Node version: 16.19.0
npm version: 8.19.3
Local ESLint version: 8.34.0
Global ESLint version: 
Operating System: linux mint

### What parser are you using?

Default (Espree)

### What did you do?

as per https://eslint.org/docs/latest/use/getting-started,
``` npm init @eslint/config``` and answered "Node" when asked whether to run in Browser or Node

### What did you expect to happen?

expected `env.node = true` in the config file

### What actually happened?

got `env.browser= true` in the config file

### Participation

- [ ] I am willing to submit a pull request for this issue.

### Additional comments

This is an easyfix when done a few times :wink: but it has been like this for a while now.

**Comment by Rec0iL99** [comment]
Hi @arnotixe, thanks for opening the issue. When running `npm init @eslint/config`, at the `Where does your code run?` step the `Browser` env is selected by default by the ESLint CLI. If you want your env to be `Node` alone, then you would have to de-select the `Browser` option by pressing the `space` key and selecting `Node` by using the same `space` key. The instructions are given along with the step in the CLI:

```bash
Where does your code run? …  (Press <space> to select, <a> to toggle all, <i> to invert selection)
```

What probably happened in your case was, you might have tried to select `Node` with the `enter` key which won't work.

Thanks for using ESLint.
